crates/macro/src/html_tree/html_iterable.rs

```diff
@@ -41,8 +41,7 @@ impl ToTokens for HtmlIterable {
         let expr = &self.0;
         let new_tokens = quote_spanned! {expr.span()=> {
             let mut __yew_vlist = ::yew::virtual_dom::VList::default();
-            let __yew_nodes: &mut ::std::iter::Iterator<Item = _> = &mut(#expr);
-            for __yew_node in __yew_nodes.into_iter() {
+            for __yew_node in ::std::iter::IntoIterator::into_iter(#expr) {
```

Comment:
If you're using it directly in a `for` loop, the call to `into_iter` is redundant, as that's already what `for` does internally.

```suggestion
            for __yew_node in #expr {
```

Reply:
Ah great point, thanks!